test/lib/pmd/PmdCatalogWrapper.test.ts

```diff
@@ -42,7 +46,7 @@ describe('PmdCatalogWrapper', () => {
 					const params = (await target.buildCommandArray())[1];
 
 					// Expect there to be 6 parameters.
-					expect(params.length).to.equal(6, 'Should have been 6 parameters');
+					expect(params.length).to.equal(6, `Should have been 6 parameters: ${params}`);
```

Comment:
This was failing before the stub of FileHandler#exists, i added some info to indicate what the contents were.

Reply:
Good find.